roles/network_plugin/canal/templates/cni-canal.conflist.j2

```diff
@@ -12,7 +12,9 @@
         "etcd_cert_file": "{{ canal_cert_dir }}/cert.crt",
         "etcd_ca_cert_file": "{{ canal_cert_dir }}/ca_cert.crt",
         "log_level": "info",
-        "log_file_path": "/var/log/calico/cni/cni.log",
+{% if calico_cni_log_file_path is defined %}
```

Comment:
```suggestion
{% if calico_cni_log_file_path %}
```
Otherwise if setting `calico_cni_log_file_path: false` like calico.md, the following line will be
```
"log_file_path": "false"
```

Reply:
absolutely. thank you for catching this.